docs/design-proposals/arch/daemonset.md

```diff
@@ -48,7 +50,21 @@ This is a variant of the above-mentioned DS in RP: cluster admin manages DaemonS
 
 This architecture mitigates the DS sync problem, but it brings more incompatibility of scale-out and scale-up, as scale-up has no such sync in the first place.
 
-### Reequired Changes
+Regarding the scope of changes to keep track of TP origin, there are options:
```

Comment:
This statement starts a new section of discussion. Please add proper section title to make it obvious.

Reply:
will make subsection